types/@metamask/obs-store.d.ts

```diff
@@ -0,0 +1,38 @@
+declare module '@metamask/obs-store' {
```

Comment:
This is temporary; we're using an older version of `@metamask/obs-store` where the types aren't generic enough.

Reply:
With the new release, I was able to remove this file.